src/actions/blockchain.js

```diff
@@ -30,39 +30,6 @@ export const setEtherTokens = createAction('SET_ETHER_TOKENS')
 
 const NETWORK_TIMEOUT = process.env.NODE_ENV === 'production' ? 10000 : 2000
```

Comment:
Do you think 10 seconds is enough? I think in production we want it to be very fast

Reply:
I didn't create this var in this PR. I think 10 Seconds until a timeout is okay, 2 seconds on dev because we usually connect to localhost